src/realmGeolBasin.ttl

```diff
@@ -96,25 +101,49 @@ soreagb:PeripheralForelandBasin rdf:type owl:Class ;
 
 ###  http://sweetontology.net/realmGeolBasin/RetroarcForelandBasin
 soreagb:RetroarcForelandBasin rdf:type owl:Class ;
-                            rdfs:subClassOf soreagb:Basin ;
+                            rdfs:subClassOf soreagb:SedimentaryBasin ;
                             rdfs:label "retroarc foreland basin"@en .
 
 
+###  http://sweetontology.net/realmGeolBasin/Subbasin
+soreagb:Subbasin rdf:type owl:Class ;
+                     dcterms:contributor <http://linked.data.gov.au/org/gsq> ;
+                     rdfs:subClassOf soreagb:SedimentaryBasin ;
+                     skos:altLabel "sub-basin"@en ;
+                     skos:definition "A sedimentary basin may be subdivided into two or more subbasins that have separate depocentres and are separated from one another by some tectonic/structural element, such as a fault or basement high. (Definition created here: Paul Blake, John McKellar)"@en ;
```

Comment:
not a definition.
suggest: a sub-basin is a sedimentary basin that is part of a larger sedimentary basin.

ScopeNote: A sedimentary basin may be subdivided into two or more subbasins that have separate depocentres and are separated from one another by some tectonic/structural element, such as a fault or basement high. (Definition created here: Paul Blake, John McKellar)